favorites.cs

```diff
@@ -0,0 +1,155 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.IO;
+using Azure.Data.Tables;
+using Azure;
+using System.Web;
+
+namespace TableStorage
+{
+    public class Favorites
+    {
+
+        string AccountName = Environment.GetEnvironmentVariable("AccountName");
+        string TableFavorite = Environment.GetEnvironmentVariable("TableFavorites");
+        string Uri = Environment.GetEnvironmentVariable("Uri");
+        string AccountKey = Environment.GetEnvironmentVariable("AccountKey");
+        private readonly TableClient _favoriteTableClient;
+
+        public Favorites() 
+        {
+            this._favoriteTableClient = 
+            new TableClient(new Uri(Uri), 
+                TableFavorite, 
+                new TableSharedKeyCredential(AccountName, AccountKey));
+        }
+
+        [FunctionName("GetFavorites")]
+        public async Task<IActionResult> RunGetFavorites(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req,
+            ILogger log)
+        {
+            try
+            {
+                Pageable<TableEntity> queryResultsFilter = _favoriteTableClient.Query<TableEntity>();
+
+                Console.WriteLine($"The query returned {queryResultsFilter.Count()} entities.");
+                
+                return new OkObjectResult(queryResultsFilter);
+
+            }
+            catch (Exception e)
+            {
+                log.LogError(e, "Problem loading");
+                return new BadRequestObjectResult("There was an issue");
+            }
```

Comment:
Its really cool to see you putting these validations in your code right away! It makes the APIs a lot nicer to work with for the other teams. (: